addon/services/page-title.js

```diff
@@ -0,0 +1,5 @@
+import Service from '@ember/service';
+
+export default class PageTitleService extends Service {
+  titleUpdated(/* newTitle */) {}
```

Comment:
Is the name okay? Got it from [this comment](https://github.com/ember-cli/ember-page-title/pull/202#issuecomment-758923682).

Reply:
What do you think about "titleDidUpdate", to leave it open for possibility of having "titleWillUpdate" in the future?

Reply:
Good suggestion, will update.

Reply:
Hum, and `titleWillUpdate` could transform the title. Good idea.

Reply:
Renamed to `titleDidUpdate`.